core/src/main/java/com/scalar/db/storage/cassandra/Cassandra.java

```diff
@@ -60,6 +60,7 @@ public Cassandra(DatabaseConfig config) {
         new TableMetadataManager(
             new CassandraAdmin(clusterManager), config.getMetadataCacheExpirationTimeSecs());
     operationChecker = new OperationChecker(metadataManager);
+    config.getDefaultNamespaceName().ifPresent(this::withNamespace);
```

Comment:
Instead of implementing this for each storage adapter, can we implement it in the parent class `AbstractDistributedStorage`?

Reply:
As discussed earlier, I moved the initialization of the default namespace in the constructor of `AbstractDistributedStorage`, `AbstractDistributedTransactionManager` and `AbstractTwoPhaseCommitTransactionManager` in https://github.com/scalar-labs/scalardb/pull/823/commits/3c012348b76479ccb7912e249287b40dce19f9c2
Please have a look again, thank you.